src/net/http_response.c

```diff
@@ -20,7 +20,7 @@ extern HttpHeader *ts_http_header_create(const char *name, size_t name_len, cons
 typedef enum HttpParseState
 {
 	HTTP_STATE_STATUS,
-	HTTP_STATE_INTERM,		/* received a single \r */
+	HTTP_STATE_INTERN,		/* received a single \r */
```

Comment:
This is quite likely wrong.

I have no idea what `interm` would represent. I'm happy to drop this change.

It might mean `interim` it might be inte... `\r` ... `\n`.